src/js/stores/MesosStateStore.js

```diff
@@ -290,19 +305,23 @@ class MesosStateStore extends GetSetBaseStore {
     });
   }
 
-  onStreamData(_state) {
-    if (!legacyUpdateTimer) {
-      this.emit(MESOS_STATE_CHANGE);
-
-      legacyUpdateTimer = setInterval(
-        () => this.emit(MESOS_STATE_CHANGE),
-        Config.getRefreshRate()
-      );
-    }
+  onStreamData() {
+    this.emit(MESOS_STATE_CHANGE);
   }
 
   onStreamError(error) {
     this.emit(MESOS_STATE_REQUEST_ERROR, error);
+
+    // This was added in the past so the stream does not swallow errors. In its
```

Comment:
🔡  This was added in the past so the stream does not swallow errors. In its
current for_m_, it causes a problem wit_h_ fixtures 1-pod and 1-empty-group and make ServiceFormModal and PodDetailpPage tests fail.

Reply:
Oh. I only checked the comments on the subscribe method(). Thanks for catching this.

Reply:
just fixed that now, that was actually much worse than the others :-)